sockets/delegateMonitor.js

```diff
@@ -309,7 +318,7 @@ module.exports = function (app, connectionHandler, socket) {
 				data.active = updateActive(res[0]);
 				data.registrations = res[1];
 				data.votes = res[2];
-				data.nextForgers = cutNextForgers(10);
+				data.nextForgers = cutNextForgers(maxLimitOfNextForgers, data.lastBlock.block.height);
```

Comment:
We shouldn't use `data.lastBlock` here because `emitData` is out of sync with `getLastBlocks`.
I suggest returning `currentBlock` on `delegates.getNextForgers` function and use it here.

Reply:
The initial idea was to re-use the data that are gathered simultaneously. Unfortunately, there is no guarantee that the data are up-to-date. I'm going to rewrite it, so the data is gathered every time an update is being sent to the browser.